app/helpers/notifier_helper.rb

```diff
@@ -2,4 +2,13 @@ module NotifierHelper
   def fp(text)
     format_paragraph(text, 72, 0)
   end
+
+  def link_to_user(display_name)
+    link_to(
+      display_name,
+      user_url(display_name, :host => SERVER_URL),
+      :target => "_blank",
+      :style => "text-decoration: none; color: #222; font-weight: bold"
+    )
+  end
 end
```

Comment:
Is there something wrong with what you've done? I mean we would normally avoid inline CSS completely but for email that's not possible so I don't see what other option there is?